server.js

```diff
@@ -85,21 +85,28 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.get(pathPrefix + 'login', login.get);
 app.post(pathPrefix + 'login', jsonParser, login.post);
 
-app.all(appPrefix + 'update_settings/*', function(req, res) {
-  // don't audit the app settings
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_revs_diff', function(req, res) {
-  // don't audit the _revs_diff
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_local/*', function(req, res) {
-  // don't audit the _local docs
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_bulk_get/*', function(req, res) {
-  // don't audit the _bulk_get request
-  proxy.web(req, res);
+// Requests we don't wish to audit
+// (usually because POST is just used here to pass more parameters)
+[
+  appPrefix + 'update_settings/*',
```

Comment:
@garethbowen do you know why we don't audit changes to settings? Is it because we know only admins can do it, or is this the standard "GETting using POST" issue?